doc/user-guide/reshaping.rst

```diff
@@ -4,7 +4,9 @@
 Reshaping and reorganizing data
 ###############################
 
-These methods allow you to reorganize your data by changing dimensions, array shape, order of values, or indexes.
+Reshaping and reorganizing data refers to the process of changing the structure or organization of data by modifying dimensions, array shapes, order of values, or indexes. Xarray provides several methods to accomplish these tasks. To convert from a Dataset to a DataArray, use the to_array() method. Unlike pandas, xarray's stack() method does not automatically drop missing values.
```

Comment:
```suggestion
Reshaping and reorganizing data refers to the process of changing the structure or organization of data by modifying dimensions, array shapes, order of values, or indexes. Xarray provides several methods to accomplish these tasks.
```
No need to mention specific method and their peculiarities in the introduction already.
I didn't check below, so make sure that the section about these methods contain the sentences you have added here.